wiki/faq.md

```diff
@@ -616,6 +628,16 @@ Read the how to help section on the [testnet wiki page](testnet "wikilink")
 Try running the client with `-rescan` or run the console command
 `repairwallet`.
 
+---
+### I get the error that the wallet is corrupt, how can I fix it? I get the error "error loading blkindex.dat"

```

Comment:
We need to make sure that it is clear that wallet.dat errors are different than blkindex.dat (blockchain database) errors.

Reply:
Fixed